cli/src/pcluster/config/common.py

```diff
@@ -276,6 +276,17 @@ def _register_validators(self):
             self._register_validator(UrlValidator, url=self.aws_batch_cli_package)
 
 
+class Imds(Resource):
+    """Represent the IMDS configuration shared between the ImageBuilder and Cluster.
+
+    It represents the Imds element that can be either at top level in the cluster config file,
+    or in the Build section of the build image config file."""
+
+    def __init__(self, require_imds_v2: bool = None, **kwargs):
+        super().__init__(**kwargs)
+        self.require_imds_v2 = Resource.init_param(require_imds_v2, default=False)
```

Comment:
Maybe I'm missing something, but why the default should be False if we would like to enforce the usage of Imds v2?

Reply:
The idea was that we will set it to `false` because we have customers that currently do IMDSv1 calls in the post install scripts, etc., so we don't want to require `IMDSv2` now, however we want to notify them that they will need to change it because we will switch the default sometime down the line.